planning/path_smoother/src/elastic_band.cpp

```diff
@@ -68,6 +68,13 @@ std::vector<double> toStdVector(const Eigen::VectorXd & eigen_vec)
 {
   return {eigen_vec.data(), eigen_vec.data() + eigen_vec.rows()};
 }
+
+std_msgs::msg::Header createHeader()
+{
+  std_msgs::msg::Header header;
+  header.frame_id = "map";
```

Comment:
Is it okay to not set the `stamp` value ?

Reply:
The created message is for visualization, so it's okay not to have a stamp.
But I agree to add. Thank you.
https://github.com/autowarefoundation/autoware.universe/pull/4371/commits/afe1e083c46b69f62f4c86bae7c8a733ecf01f0f